[3866] Link to acknowledgements on about page

rart opened the pull request:
https://github.com/craftercms/craftercms/issues/3866

* Adds acknowledgements statement to the about page
* Internationalization of the acknowledgement message
* Adds the `entitlementValidator` for future usage to entry.ftl

rart committed:
[3866] Link to acknowledgements on about page
https://github.com/craftercms/craftercms/issues/3866

* Adds acknowledgements statement to the about page
* Internationalization of the acknowledgement message
* Adds the `entitlementValidator` for future usage to entry.ftl

rart committed:
Makes the attributions page open in a new tab.

rart committed:
Update attribution prepend

rart committed:
Update attribution append

rart committed:
Update attribution

rart committed:
Update attribution

rart committed:
Update entry.ftl

rart committed:
Merge branch 'develop' of https://github.com/craftercms/studio-ui into feature/3866

rart committed:
Merge branch 'feature/3866' of https://github.com/rart/studio-ui into feature/3866

sumerjabri commented on static-assets/ng-views/about.html:
```diff
@@ -15,20 +15,20 @@
+          <p><strong>{{ 'about.STUDIO_VERSION_NUMBER' | translate }}</strong>: {{versionNumber}}</p>
+          <p><strong>{{ 'about.BUILD_NUMBER' | translate }}</strong>: {{aboutStudio.packageBuild}}</p>
+          <p><strong>{{ 'about.BUILD_DATE_TIME' | translate }}</strong>: {{aboutStudio.packageBuildDate | date:'MM-dd-yyyy HH:mm:ss Z'}}</p>
+          <p>{{ 'about.ATTRIBUTION_PREPEND' | translate }} <a target="_blank" href="https://docs.craftercms.org/en/3.1/acknowledgements/index.html">open source software</a>.</p>
```
It needs to read:
```
Crafter CMS is made possible by these other open source software projects.
```

Right now I'm seeing:
```
Crafter CMS is made possible by these other open source software.
```

sumerjabri reviewed (changes requested):
See my note.

rart committed:
[3866] Link to acknowledgements page

https://github.com/craftercms/craftercms/issues/3866

Uses product information API (version) to dynamically generate the docs & acknowledgements link
Updated translations to match desired copy
Adds angular minified versions for faster production load
Adds angular-sanitize-1.5.0.js, needed for innerHTML setting
Renames angular.js to angular-1.5.0.js
Moved related translations to the codebase next system and removed from legacy system locale files

rart committed:
Improve styling of "about" view heading.
